### ISSUE TITLE
One case prediction example

### ISSUE THREAD

**Issue by GravityPHY** [mcarroll8/ISPIPab#1]
Hi there,

I am trying to use your method to test epitope prediction on one antigen structure. I only have the structure in pdb format. Can you help provide an example how the input file should look like in your program? I tried make the input file path to pdb but it does not work.

Thanks!
GravityPHY

**Comment by nick-youngblut** [comment]
Yeah, it's not clear on how to actually generate the input. The README states:

> -i: [str] default: input.csv - CSV Filename with columns: "residue","predus","ispred","dockpred","annotated". The column residue is of the form {residue number}_{PDB ID}.{chain}. The annotated column is 1 or interface residue and 0 for non-interface residue

...but how is that CSV generated?

@mcarroll8 how did you generate [SPPIDER_ISPRED_DOCKPRED_classifier_ispip_input.csv](https://github.com/mcarroll8/ISPIPab/blob/main/ISPIPab_Prediction_Scripts/Data/input/SPPIDER_ISPRED_DOCKPRED_classifier_ispip_input.csv)? Can you please share your code?
